jasmine-standalone-3.5.0/src/tenPinController.js

```diff
@@ -0,0 +1,50 @@
+
+
+$(document).ready(function(){
+    bowling = new Bowling();
+
+    arr = ["#1-1", "#1-2", "#1-3", "#1-4" ,"#1-5", "#1-6", "#1-7", "#1-8", "#1-9","#1-10","#1-11"]
+    elm = 0; 
+    prv = 0;
+    two = false
+
+    $("#board").keyup(function(event) {
+        
+       if (two == false && event.keyCode == 13){
+             two = true
+             prv = parseInt($(arr[elm]).val()) 
+             $(arr[elm]).val("");
+
+                if (prv == 10){
+                     bowling.calcPoints(10,0)
+                     $("#the").text(bowling.player_score); 
+                     $(arr[elm]).replaceWith(function() {
+                    return '<div>' + bowling.player_score +  '</div>';  
+            })
+           
+            elm++  
+            two = false
```

Comment:
I think elm is a bit of a dodgy name here, could do with something being clearer